incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -499,6 +503,7 @@ func (r *HierarchyReconciler) writeNamespace(ctx context.Context, log logr.Logge
 		return false, nil
 	}
 
+	stats.Stats.HcWrite.Incr()
```

Comment:
Sure, it was tracked separately in the previous commit and here I put them together just because the writes both happen in the HC reconciliation.

Separating them sounds more reasonable!

Reply:
I think there will be many cases where we write one and not the other, though this isn't true for reads!